scripts/west_commands/runners/openocd.py

```diff
@@ -66,8 +67,12 @@ def __init__(self, cfg, pre_init=None, reset_halt_cmd=DEFAULT_OPENOCD_RESET_HALT
         self.gdb_cmd = [cfg.gdb] if cfg.gdb else None
         self.tui_arg = ['-tui'] if tui else []
         self.halt_arg = [] if no_halt else ['-c halt']
+        self.no_init = [] if no_init else ['-c init']
+        self.no_targets = [] if no_targets else ['-c targets']
         self.serial = ['-c set _ZEPHYR_BOARD_SERIAL ' + serial] if serial else []
         self.use_elf = use_elf
+        self.gdb_init = gdb_init
+        self.no_load = [] if no_load else ['-ex', 'load']
```

Comment:
Could you remove the `no_` prefix for these variables? Just like `halt_arg` above. How these are being used is actually the opposite of what the names suggest.

Reply:
@dcpleung I updated it to folllow `no_halt` argument and `halt_args` variable.